tests/pagination/src/commonMain/graphql/schema.graphqls

```diff
@@ -0,0 +1,10 @@
+type Query @fieldPolicy(forField: "users", keyArgs: "id") {
```

Comment:
That `@fieldPolicy` is unused here?

Reply:
You're right 🤦‍♀️. It's not even correct (no `id` arg on `users`).